src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -272,6 +272,7 @@
   <ItemGroup>
     <VsixIncludeFile Include=".vsixinclude" />
     <VsixIgnoreFile Include=".vsixignore" />
+    <VsixIgnoreFile Include=".vsixignore.CI" Condition="'$(IsCIBuild)' == 'true'" />
```

Comment:
I'd just like to confirm if someone downloaded the vsix from the dev branch, and from this branch, and validated that the files in both vsix's are the same?

Reply:
I did, and they are not. See these commits:
![image](https://github.com/user-attachments/assets/3eb3c60a-47f2-45bc-bc9e-2cfb87b52774)

Reply:
Is the PR ready to be reviewed and merged, or is it still being worked on while the file differences are still being worked on? 

When this PR pops up in my notification list, I'm not sure if I should review and approve, or if I should wait.

Reply:
It is ready for review. Sorry for the noise in the past day or so, we kept thinking it was done but little things kept cropping up. It appears my build is green which is why I'm updating the description now.

Reply:
I added a nifty table.

Reply:
> When this PR pops up in my notification list, I'm not sure if I should review and approve, or if I should wait.

Open to suggestions. If I mark it as draft, you'll still get those notifications as GitHub does not unsubscribe anyone.

Reply:
yes, but if I see a notification for a draft PR, I can immediately move on, knowing I'll get another notification when it's ready for review, at which time I should look at the diff.